definitions/3.0.0/channel.json

```diff
@@ -7,8 +7,12 @@
     }
   },
   "properties": {
-    "$ref": {
-      "$ref": "http://asyncapi.com/definitions/3.0.0/ReferenceObject.json"
```

Comment:
For the reference, this change will solve https://github.com/asyncapi/spec/issues/699 if I'm correct, please confirm @char0n

Reply:
@smoya yes I can confirm.